### ISSUE TITLE
About The Essay

### ISSUE THREAD

**Issue by HZIH** [yuankaishen2001/AUFormer#2]
Thanks to the authors for their work, there are some question about the essay:
1. Which checkpoint do you select for the Baseline Vit-B, I try the checkpoint pretrained in ImageNet( just the same in your essay), but the result is relatively low, lower than 60, about 50-55 F1 Score.

2. In the description about the "CA Operator", you mention that "Q is Hadamard producted with the vectors of K in its neighborhood (assuming a size of S ×S)" to obtain the correlation matrix. However, hadamard product needs two vectors to be the same dimension. If Q's dimension is "Batchsize × N × C", then K's Dimension should be "Batchsize × (N × S × S) × C" after the operation done in the essay. This results in a conflict of the calculation of hadamard product. Moreover, Hadamard product seems not suitable for correlation matrix's estimation, and dot product may be a better choice.

3. What's the main difference among your loss function, WA Loss[2] and Asymmetric Loss[1]? We can find that WA Loss is a variant of Asymmetric Loss by adding AUs' weights and setting gamma_pos to 0, gamma_neg to 1 and m to be 0. Your loss function MDWA seems no difference between WA Loss and Asymmetric Loss, even for the gradient analysis, it's the same to the Asymmetric Loss.

I will appreciate that if the authors can reponse to my questions, and show me the code to further let me know the details of the architecture described in the essay. 

[1] Ridnik T, Ben-Baruch E, Zamir N, et al. Asymmetric loss for multi-label classification[C]//Proceedings of the IEEE/CVF international conference on computer vision. 2021: 82-91.
[2] Luo C, Song S, Xie W, et al. Learning multi-dimensional edge feature-based au relation graph for facial action unit recognition[J]. arXiv preprint arXiv:2205.01782, 2022.
